ProtocolTests/Protocol/Compliancies/MinimumRequiredVersion/CheckMinimumRequiredVersionTag/Samples/Codefix/BelowMinimumSupportedVersion.xml

```diff
@@ -0,0 +1,5 @@
+﻿<Protocol xmlns="http://www.skyline.be/validatorProtocolUnitTest">
+    <Compliancies>
+        <MinimumRequiredVersion>10.1.0.0 - 9966</MinimumRequiredVersion>
```

Comment:
Will this not break when DIS changes what it knows as the minimumRequiredVersion?

Reply:
Current minimum required version is 10.2. This is a default value specified in an internal constructor for the ValidatorSettings which is used for unit tests (also mentioned in xml doc of that constructor).